pages/checkout/access.tsx

```diff
@@ -57,55 +57,69 @@ const CheckoutAccessView: NextPage = () => {
 
   const onContinueClick = () => {
     dispatch.checkout.setStep(2)
-    router.push("/checkout/confirm")
+    router.push({
+      pathname: "/checkout/confirm",
+      query: { id: assetId },
+    })
   }
 
   return (
     <>
       <Container>
-        <CheckoutWizard>
-          <Headline>
-            {intl.formatMessage({ id: "checkout.access.headline" })}
-          </Headline>
-          <Ingress>
-            {intl.formatMessage({ id: "checkout.access.ingress" })}
-          </Ingress>
-
-          <ExampleContainer>
-            <Typography variant="caption">
-              {intl.formatMessage({ id: "checkout.access.exampleLabel" })}
-            </Typography>
-            <Typography variant="caption">
-              {intl.formatMessage({ id: "checkout.access.exampleBody" })}
-            </Typography>
-          </ExampleContainer>
-
-          <TextFieldContainer>
-            <TextField
-              multiline
-              id="desciption"
-              label={intl.formatMessage({ id: "checkout.access.descriptionInput.label" })}
-              placeholder={intl.formatMessage({ id: "checkout.access.descriptionInput.placeholder" })}
-              onChange={onDescriptionChange}
-            />
-          </TextFieldContainer>
-
-          <ButtonContainer>
-            <Button
-              variant="outlined"
-              color="secondary"
-            >
-              {intl.formatMessage({ id: "common.cancel" })}
-            </Button>
-            <Button disabled={description.length < 10} onClick={onContinueClick}>
-              {intl.formatMessage({ id: "common.continue" })}
-            </Button>
-          </ButtonContainer>
+        <CheckoutWizard assetId={assetId}>
+          {!assetId ? <NoAsset />
```

Comment:
maybe we could move this inside `CheckoutWizard` so we don't have to do this check on every checkout view?

Reply:
Yup - definitely! But I just wanted to figure out how we should pass the asset id and who is in charge of it first, so stupid duplication for now